tests/test_create_pod_attaching_multiple_pvc_start_IO.py

```diff
@@ -0,0 +1,376 @@
+        except exceptions.CommandFailed:
+            log.info(f'Exception PVC {i} is not deleted')
+            return False
+
+
+def delete_storage_class():
+    """
+    Delete the Storage class
+
+    """
+    template = os.path.join(TEMPLATES_DIR, "storageclass.yaml")
+    log.info(f"Delete the PVC")
+    data = templating.generate_yaml_from_jinja2_template_with_data(
+        template
+    )
+    sc_name = data['metadata']['name']
+    with open(TEMP_YAML, 'w') as yaml_file:
+        yaml.dump(data, yaml_file)
+    try:
+        SC.delete(yaml_file=TEMP_YAML)
+        open(TEMP_YAML, 'w').close()
+        return True
+    except exceptions.CommandFailed:
+        log.info(f"Exception SC {sc_name} is not deleted")
+        return False
+
+
+def delete_pool():
+    """
+    Delete the pool
+
+    """
+    template = os.path.join(TEMPLATES_DIR, "CephBlockPool.yaml")
+    log.info(f"Delete the PVC")
+    data = templating.generate_yaml_from_jinja2_template_with_data(
+        template,
+    )
+    pool_name = data['metadata']['name']
+    with open(TEMP_YAML, 'w') as yaml_file:
+        yaml.dump(data, yaml_file)
+    try:
+        CBP.delete(yaml_file=TEMP_YAML)
+        open(TEMP_YAML, 'w').close()
+        return True
+    except exceptions.CommandFailed:
+        log.info(f"Exception CephBlockPool {pool_name} not deleted")
+        return False
+
+
+def delete_secret():
+    """
+    Delete the Secret
+
+    """
+    template = os.path.join(TEMPLATES_DIR, "secret.yaml")
+    log.info(f"Delete the PVC")
+    data = templating.generate_yaml_from_jinja2_template_with_data(
+        template
+    )
+    secret_name = data['metadata']['name']
+    with open(TEMP_YAML, 'w') as yaml_file:
+        yaml.dump(data, yaml_file)
+    try:
+        SEC.delete(yaml_file=TEMP_YAML)
+        open(TEMP_YAML, 'w').close()
+        return True
+    except exceptions.CommandFailed:
+        log.info(f"Exception Secret {secret_name} not deleted")
+        return False
+
+
+@tier1
+@pytest.mark.mine
+@pytest.mark.usefixtures(
+    test_fixture.__name__,
+)
+class TestCaseOCS260(ManageTest):
+    """
+    Create a pod attached with multiple PVC and run IO
+    https://polarion.engineering.redhat.com/polarion/#/project
+    /OpenShiftContainerStorage/workitem?id=OCS-260
+
+    """
+    pvc_count = 10  # create 10 PVC to be mounted in POD
+    io_duration_sec = 180  # amount of time IO needs to executed
+    pv_list = []
+    cirros_name = []
+
+    def test_run_ocs_260(self):
+        """
+        Test case OCS-260
+
+        """
+        for i in range(self.pvc_count):
+            self.pv_list.append('rbd-pvc' + str(i))
+            create_persistent_volume(self.pv_list[i])
+            log.info(f"pvc {self.pv_list[i]} created success")
+        self.cirros_name.append(create_pod(self.pv_list))
+        log.info(f"Cirros pod name {self.cirros_name[0]}")
+        log.info(f"Running IO's for {self.io_duration_sec} seconds")
```

Comment:
this is still not implemented.
Please add:
# TODO

Reply:
Cirros yaml defaults creates the pod with pvc mount and starts DD io, which is part of the cirros yaml file. Added the IO duration to hold on the cleanup process.